os/x11/window.h

```diff
@@ -128,6 +130,8 @@ class WindowX11 : public Window {
   bool m_resizable = false;
   bool m_transparent = false;
 
+  XEvent m_prev_key_event = { 0 };
```

Comment:
As suggested in another comment, I think we should change this to KeySym m_pressedKeySym = 0;

Reply:
Done.